### PR TITLE
SQL API: Add support for simple aggregation queries

### PR DESCRIPTION
**Pull request by AdityaGoel11**
- Creates a Logical Aggregate operator.
- Creates an Aggregate Operator that can handle simple SQL aggrgation queries.
- Precisely, the query can have 1 aggregation field, 0 or 1 grouping field, and SUM, MIN, MAX, COUNT, or AVG as the aggregate function.

### CODE REVIEW

File: wayang-api/wayang-api-sql/src/test/java/org/apache/wayang/api/sql/SqlTest.java
Comment (kbeedkar) [inline wayang-api/wayang-api-sql/src/test/java/org/apache/wayang/api/sql/SqlTest.java:1]: @AdityaGoel11 please undo the changes to this file. This test class has nothing to do with SQL queries with aggregations.

File: wayang-api/wayang-api-sql/src/test/resources/model.json [hunk 1]
```diff
@@ -1,24 +1,16 @@
...
-    },
-    {
-      "name": 'fs',
-      "type": 'custom',
-      "factory": 'org.apache.calcite.adapter.file.FileSchemaFactory',
-      "operand": {
-        "directory": 'C:/incubator-Wayang-CrossPlatform/incubator-wayang-SQL/wayang-api/wayang-api-sql/src/test/resources'
+      "jdbcDriver": "org.postgresql.Driver",
+      "jdbcUrl": "jdbc:postgresql://localhost:5432/dvdrental",
+      "jdbcUser": "aditya",
```
Comment (kbeedkar) [inline wayang-api/wayang-api-sql/src/test/resources/model.json:12]: @AdityaGoel11 Do not delete the fs schema, it is meant to support SQL queries over csv files.

**Review by kbeedkar — CHANGES_REQUESTED** [review]
@AdityaGoel11 please see my reviews.

**Comment by AdityaGoel11** [comment]
> @AdityaGoel11 please see my reviews.

Thank you for the review, sir. I have reverted all the changes made to SqlTest.java file and reinserted the fs schema in model.json file.

**Review by kbeedkar — APPROVED** [review]
Looks good.

**Comment by kbeedkar** [comment]
Thanks for the initial contribution @AdityaGoel11. Looking forward to more complete support for aggregate queries.
